qcfractal/interface/models/records.py

```diff
@@ -428,8 +428,11 @@ class OptimizationRecord(RecordBase):
         ...,
         description="The ID of the molecule which was passed in as the reference for this Optimization"
     )
-    qc_spec: QCSpecification
-    keywords: Dict[str, Any] = Schema(  # TODO: defined in Base
+    qc_spec: QCSpecification = Schema(
+        ...,
+        description="The specification of the quantum chemistry calculation to run at each point."
+    )
+    keywords: Dict[str, Any] = Schema(
```

Comment:
A few examples where we need to rewrite the doc strings. We have: "What type of calculation this result is tied to."

Where in `common_models.py` I rewrote them to use declarative voice: "The type of calculation that is being performed (e.g., energy, gradient, Hessian, ...)."

We should try to keep the strings consistent as well, like in `common_models.py` we have specific phrasing for version which we should extend to this file.

For Enum's we could either change our documentation tech to take the Enum doctoring or we could do:

```
    driver: DriverEnum = Schema(
        ...,
        description= DriverEnum.__doc__
    )
```

I do like being explicit about this here, by calling the `__doc__` attribute.